.env.production

```
@@ -0,0 +1,25 @@
+VERCEL_GIT_COMMIT_AUTHOR_LOGIN=""
+VERCEL_GIT_COMMIT_AUTHOR_NAME=""
+VERCEL_GIT_COMMIT_MESSAGE=""
+VERCEL_GIT_COMMIT_REF=""
+VERCEL_GIT_COMMIT_SHA=""
+VERCEL_GIT_PREVIOUS_SHA=""
+VERCEL_GIT_PROVIDER=""
+VERCEL_GIT_PULL_REQUEST_ID=""
+VERCEL_GIT_REPO_ID=""
+VERCEL_GIT_REPO_OWNER=""
+VERCEL_GIT_REPO_SLUG=""
+VERCEL_OIDC_TOKEN="eyJhbGciOiJSUzI1NiIsInR5cCI6IkpXVCIsImtpZCI6Im1yay00MzAyZWMxYjY3MGY0OGE5OGFkNjFkYWRlNGEyM2JlNyJ9.eyJpc3MiOiJodHRwczovL29pZGMudmVyY2VsLmNvbS9tYXhpbS1hLXMtcHJvamVjdHMiLCJzdWIiOiJvd25lcjptYXhpbS1hLXMtcHJvamVjdHM6cHJvamVjdDp5b211OmVudmlyb25tZW50OmRldmVsb3BtZW50Iiwic2NvcGUiOiJvd25lcjptYXhpbS1hLXMtcHJvamVjdHM6cHJvamVjdDp5b211OmVudmlyb25tZW50OmRldmVsb3BtZW50IiwiYXVkIjoiaHR0cHM6Ly92ZXJjZWwuY29tL21heGltLWEtcy1wcm9qZWN0cyIsIm93bmVyIjoibWF4aW0tYS1zLXByb2plY3RzIiwib3duZXJfaWQiOiJ0ZWFtX3VVWTdXVXNLNFFSeGhFTEZZT1hHYXF1cCIsInByb2plY3QiOiJ5b211IiwicHJvamVjdF9pZCI6InByal94WUJYaEdWSGU2aWp1MVRmQnFGMm5UVGY4YVoxIiwiZW52aXJvbm1lbnQiOiJkZXZlbG9wbWVudCIsInBsYW4iOiJob2JieSIsInVzZXJfaWQiOiJ0YWhEYUZGTDlBNHY4ZGk1VjhRU1htckYiLCJjbGllbnRfaWQiOiJjbF9IWXlPUEJOdEZNZkhoYVVuOUw0UVBmVFp6NlRQNDdicCIsIm5iZiI6MTc3OTMxMjYyMywiaWF0IjoxNzc5MzEyNjIzLCJleHAiOjE3NzkzNTU4MjN9.XnXQL5pU_U8MiCYeG3fC0fGD8-7LZFmFx8K9yQHvOBUfYEE7hhYU27erlbmVyDbUh9pL78aYSuU86UpqbUsll4iSHMmZ_eROUtEwTa2CZIhHbDEzMmJHWQbdN2J3dIM9YZUEaaOBmaIsc-wFedLsMBr6vG6zoHDGjeApbLF07U67ZAzLhMMyziwNEwf_3un4FOjBEgbIzrWMd_HUuv4351GAjQHaLcK4kGNmrVkwffeC5Yykg8XqpALwPmz_J0CsOJ0iWp4yS7cpbKMVhmk0twUoMS5cEVDvkTngdZ8b9Ql7FCqIrH8omTcyYv2a2Otxj9GdBVqwNJpIQ6Nz3khMrg"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)